src/components/Publish/_types.ts

```diff
@@ -61,3 +73,23 @@ export interface MetadataAlgorithmContainer {
   tag: string
   checksum: string
 }
+
+export type AlgorithmConsumerParameterType =
```

Comment:
issue: should be renamed to e.g. `ConsumerParameterType`, since it is also used in services now.